.github/workflows/integrity-check.yml

```diff
@@ -56,5 +56,30 @@ jobs:
       # runs code coverage badge generation, throws if README output differs to README to be checked in (thus detects if tests were not run)
       - run: npm run make-coverage-badges:ci
 
+  test-with-browsers:
+    # Run browser tests using macOS so that WebKit tests don't fail under a Linux environment
+    runs-on: macos-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
```

Comment:
@flothjl, for my learning: why are commit hashes used instead of referencing the friendlier version tags? Also why not go all the way and use latest available?

Reply:
@thehenrytsai this was me trying to follow similar patterns to other areas in TBD that have browser tests in their CI. Namely, web5-js. https://github.com/TBD54566975/web5-js/blob/main/.github/workflows/tests-ci.yml#L77
Let me know, happy to update this to latest, just wasn't sure what the preference was :).

Reply:
Time to consult the guru why commit hash is deliberately used, @frankhinek?

Reply:
@flothjl, we don't have to wait on @frankhinek, if you update the reference to latest by version tag, then we can just get this approved and merged.

We can always change if this turns out be a stupid thing to do!